openfl/utilities/workspace.py

```diff
@@ -55,10 +56,15 @@ def _install_requirements(self):
         else:
             logger.error('No ' + requirements_filename + ' file found.')
 
+    def _rm_readonly_dir(self, func, path, _):
+        """Error handler for shutil.rmtree for clearing the readonly bit."""
+        os.chmod(path, stat.S_IWRITE)
+        func(path)
```

Comment:
Maybe make sense move this function as standalone to utils because this is not ExperimentWorkspace specific functionality.